src/main/resources/elasticsearch/index/instance.json

```diff
@@ -7,6 +7,12 @@
     "mapping.total_fields.limit": 250
   },
   "analysis": {
+    "char_filter": {
+      "remove_hyphen_char_filter": {
```

Comment:
Isn't we already remove the hyphens in the processor?

Reply:
it is used to remove hyphens from incoming request also